support/kernel-config/configuration.nix

```diff
@@ -465,7 +465,7 @@ in
 
     # AArch64 specifics
     (mkIf isAarch64 (helpers: with helpers; mkDefaultIze {
-      ARM64_SME = whenAtLeast "5.19" yes;
+      ARM64_SME = no;
```

Comment:
Let's add the rationale as one sentence, and then the link you shared in the PR. Just in case the next person looking at it isn't looking at the (good!) commit message.

Reply:
I added a comment to the code. Is the explanation sufficient?